figure_generation/manuscript_figure_generation.py

```diff
@@ -0,0 +1,273 @@
+# Idea here is that word2vec generates words arbitrarily and direct comparison is meaningless without an alignment factor.
+# 1. Panel A - Show umap plot of word models without alignment
+# 2. Panel B - Show umap plot of the same word models with alignment
+# 3. Panel C - Show same umap plot but with two examples that show intra year variability (show the opacity plot with word such as 'of', 'interleukin-18', 'pandemic')
+
+umap_visualization_path = Path(
+    "../multi_model_revamp/output/figure_data_and_figures/alignment_visualization"
+)
+
+# +
+panel_one = sg.fromfile(umap_visualization_path / "unaligned_2010_plot.svg")
+
+# Convert pt units to pixel units
+# Vince's tutorial FTW
+panel_one_size = (
+    np.round(float(panel_one.root.attrib["width"][:-2]) * 1.33, 0),
+    np.round(float(panel_one.root.attrib["height"][:-2]) * 1.33, 0),
+)
+
+scale_x = 1.4
+scale_y = 1.4
+
+print(f"original: {panel_one_size}")
+print(f"scaled:{(panel_one_size[0]*scale_x,panel_one_size[1]*scale_y)}")
+
+panel_one = panel_one.getroot()
+panel_one.scale(x=scale_x, y=scale_y)
+panel_one.moveto(50, 50)
+
+# +
+panel_two = sg.fromfile(umap_visualization_path / "unaligned_2010_probiotics_plot.svg")
+
+panel_two_size = (
+    np.round(float(panel_two.root.attrib["width"][:-2]) * 1.33, 0),
+    np.round(float(panel_two.root.attrib["height"][:-2]) * 1.33, 0),
+)
+scale_x = 1.4
+scale_y = 1.4
+
+print(f"original: {panel_two_size}")
+print(f"scaled:{(panel_two_size[0]*scale_x, panel_two_size[1]*scale_y)}")
+
+panel_two = panel_two.getroot()
+panel_two.scale(x=scale_x, y=scale_y)
+panel_two.moveto(954, 50)
+
+# +
+panel_three = sg.fromfile(umap_visualization_path / "aligned_2010_probiotics_plot.svg")
+
+panel_three_size = (
+    np.round(float(panel_three.root.attrib["width"][:-2]) * 1.33, 0),
+    np.round(float(panel_three.root.attrib["height"][:-2]) * 1.33, 0),
+)
+scale_x = 1.4
+scale_y = 1.4
+
+print(f"original: {panel_three_size}")
+print(f"scaled:{(panel_three_size[0]*scale_x, panel_three_size[1]*scale_y)}")
+
+panel_three = panel_three.getroot()
+panel_three.scale(x=scale_x, y=scale_y)
+panel_three.moveto(50, 631)
+# -
+
+panel_one_label = sg.TextElement(30, 30, "A", size=30, weight="bold")
+panel_two_label = sg.TextElement(954, 30, "B", size=30, weight="bold")
+panel_three_label = sg.TextElement(30, 631, "C", size=30, weight="bold")
+
+# +
+figure_one = sg.SVGFigure(Unit(1650), Unit(1200))
+
+figure_one.append(
+    [
+        etree.Element("rect", {"width": "100%", "height": "100%", "fill": "white"}),
+        panel_one,
+        panel_two,
+        panel_three,
+        panel_one_label,
+        panel_two_label,
+        panel_three_label,
+    ]
+)
+display(SVG(figure_one.to_str()))
+# -
+
+# save generated SVG files
+figure_one.save("output/Figure_1.svg")
+svg2png(bytestring=figure_one.to_str(), write_to="output/Figure_1.png", dpi=600)
+
+os.system(
+    "convert -compress LZW -alpha remove output/Figure_1.png output/Figure_1.tiff"
+)
```

Comment:
Do you need the `os.system` call here? I think it should work in a Jupyter notebook to just do this:
```suggestion
!convert -compress LZW -alpha remove output/Figure_1.png output/Figure_1.tiff
```

Reply:
Ah! I forgot you can do that with jupyter. Yeah, I don't need `os.system`.